mcf-browser/src/back-end/utils.js

```diff
@@ -61,13 +70,12 @@ async function getRemotePropertyLabels(dcid) {
 async function getRemotePropertyValues(dcid, label, isInverse) {
   const direction = isInverse ? 'in' : 'out';
   const targetUrl =
-      (API_ROOT + '/node/property-values?limit=500&dcids=' + dcid +
-       '&property=' + label + '&direction=' + direction);
+      (API_ROOT + '/v1/property/' + direction + '/' + label +
```

Comment:
We're updating the v1 REST API signature to /v1/property/values/{direction}/{dcid}/{property}
@shifucun to confirm: We should update to the new signature now, right?

Reply:
Do you know when the new signature update is happening? Last time I spoke to @shifucun, I think he said to keep using the old signature for now.

Reply:
Just updated the v1 API, please use the one Julia suggested.

Reply:
Spoke to Bo and the new signature is live now, the next commit will have this change!